x-pack/test/plugin_api_integration/test_suites/task_manager/check_registered_task_types.ts

```diff
@@ -0,0 +1,128 @@
+        'actions:.server-log',
+        'actions:.servicenow',
+        'actions:.servicenow-itom',
+        'actions:.servicenow-sir',
+        'actions:.slack',
+        'actions:.swimlane',
+        'actions:.teams',
+        'actions:.webhook',
+        'actions:.xmatters',
+        'actions_telemetry',
+        'alerting:.es-query',
+        'alerting:.geo-containment',
+        'alerting:.index-threshold',
+        'alerting:apm.anomaly',
+        'alerting:apm.error_rate',
+        'alerting:apm.transaction_duration',
+        'alerting:apm.transaction_error_rate',
+        'alerting:logs.alert.document.count',
+        'alerting:metrics.alert.anomaly',
+        'alerting:metrics.alert.inventory.threshold',
+        'alerting:metrics.alert.threshold',
+        'alerting:monitoring_alert_cluster_health',
+        'alerting:monitoring_alert_cpu_usage',
+        'alerting:monitoring_alert_disk_usage',
+        'alerting:monitoring_alert_elasticsearch_version_mismatch',
+        'alerting:monitoring_alert_jvm_memory_usage',
+        'alerting:monitoring_alert_kibana_version_mismatch',
+        'alerting:monitoring_alert_license_expiration',
+        'alerting:monitoring_alert_logstash_version_mismatch',
+        'alerting:monitoring_alert_missing_monitoring_data',
+        'alerting:monitoring_alert_nodes_changed',
+        'alerting:monitoring_alert_thread_pool_search_rejections',
+        'alerting:monitoring_alert_thread_pool_write_rejections',
+        'alerting:monitoring_ccr_read_exceptions',
+        'alerting:monitoring_shard_size',
+        'alerting:siem.eqlRule',
+        'alerting:siem.indicatorRule',
+        'alerting:siem.mlRule',
+        'alerting:siem.newTermsRule',
+        'alerting:siem.notifications',
+        'alerting:siem.queryRule',
+        'alerting:siem.savedQueryRule',
+        'alerting:siem.thresholdRule',
+        'alerting:transform_health',
+        'alerting:xpack.ml.anomaly_detection_alert',
+        'alerting:xpack.ml.anomaly_detection_jobs_health',
+        'alerting:xpack.uptime.alerts.durationAnomaly',
+        'alerting:xpack.uptime.alerts.monitorStatus',
+        'alerting:xpack.uptime.alerts.tls',
+        'alerting:xpack.uptime.alerts.tlsCertificate',
+        'alerting_health_check',
+        'alerting_telemetry',
+        'alerts_invalidate_api_keys',
+        'apm-telemetry-task',
+        'cases-telemetry-task',
+        'cleanup_failed_action_executions',
+        'cloud_security_posture-stats_task',
+        'dashboard_telemetry',
+        'endpoint:metadata-check-transforms-task',
+        'endpoint:user-artifact-packager',
+        'fleet:reassign_action:retry',
+        'fleet:unenroll_action:retry',
+        'fleet:update_agent_tags:retry',
+        'fleet:upgrade_action:retry',
+        'osquery:telemetry-configs',
+        'osquery:telemetry-packs',
+        'osquery:telemetry-saved-queries',
+        'report:execute',
+        'reports:monitor',
+        'sampleOneTimeTaskTimingOut',
```

Comment:
There are some test task types included in the response, I was wondering if anyone has ideas on how to handle them. I was thinking maybe we could change the tests to prefix them with `test:`

Reply:
ya, that sounds like a great idea!  Doesn't need to be in this PR, but it may be very easy ...

Reply:
Sure that sounds good!

Reply:
Resolved here 27b932095abd7fb6f12dfd6b43b38fdea3a43d09